app/components/comments.js

```diff
@@ -1,30 +1,34 @@
+import {inject as service} from '@ember/service';
 import Component from '@ember/component';
 import {task} from 'ember-concurrency';
-import {inject as service} from '@ember/service';
+import {tracked} from '@glimmer/tracking';
 
-export default Component.extend({
-  store: service(),
```

Comment:
Add a tagName so we can see the @tagName decorator and mention other related decorators